greenpithumb/sleep_window_parser.py

```diff
@@ -0,0 +1,26 @@
+"""Parses sleep window strings."""
```

Comment:
Suggests naming this file `sleep_windows.py`.

`sleep_windows.parse` is clear and avoids the stutter-naming of "parser.parse", plus if we later decide to represent sleep windows differently, it'll be easy to add something here like `sleep_windows.Window`